container/cnap-inference/Dockerfile

```diff
@@ -30,11 +12,7 @@ ARG pip_mirror
 
 COPY ./cnap /cnap
 
-COPY --from=downloader \
-    /public/ssd_mobilenet_v1_coco/ssd_mobilenet_v1_coco_2018_01_28/frozen_inference_graph.pb \
-    /demo/model/model.pb
-
-RUN chown -R appuser:appuser /demo /cnap
+RUN chown -R appuser:appuser /cnap
```

Comment:
@Hulongyin I'm not sure about removing the `/demo` directory, please help confirm this change.

Reply:
Ok, the `/demo` directory in inference service was used to store the models, but now it's not needed, so it's OK to remove it.